portfolio/src/main/webapp/script.js

```diff
@@ -260,10 +260,16 @@ function createListElement(comment) {
   return liElement;
 }
 
+function hideCommentsForm() {
+    var form = document.getElementById("comments-form");
+    form.style.display = "none";
+}
+
 /**
  * Actions to run once webpage has loaded
  */
 function onLoad() {
+    hideCommentsForm(); //should be defaulting from css - this is a double check
```

Comment:
Is it necessary? Seems like a workaround for something not quite working.

Maybe some sort of sort of assertion is more appropriate to let you know if this is happening, ie is the commentForm not hidden, then log to console or throw exception.

Reply:
No, not necessary, and also not present in the final commit. Sorry, must have commited remnants of my debugging code at one point.